main.c

```diff
@@ -258,7 +264,7 @@ void read_info(void)
 	ssize_t i, n;
 	char buf[BAR_L_LEN];
 
-	while (true) {
+	for (;;) {
```

Comment:
Why change this, looks like we're using `while (true)` on other places too? I think the PR is about making current style consistent, not changing it.

Reply:
Looks like we got 3 `while (true)` and 3 `for (;;)`.
Personally I like `while (true)` better, but I guess this is fine too.